### ISSUE TITLE
Review Draft Proposal

### ISSUE THREAD

**Issue by rashidahamedmeeran** [RnDProjectsDeebul/RashidTajdeenRnD#1]

**Comment by rashidahamedmeeran** [comment]
[Rnd_Rashid_Tajdeen.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/8641699/Rnd_Rashid_Tajdeen.pdf)

**Comment by rashidahamedmeeran** [comment]
[Rnd_Rashid_Tajdeen.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/8904200/Rnd_Rashid_Tajdeen.4.pdf)

Added a Draft proposal

**Comment by deebuls** [comment]
First of all good attempt. First usable output you have produced, so be proud of your self. 
Please check the attachment with the review comments in detail. . Please don't   work on all of of them now, Just take a look and understand the issues in the writing. For next iteration focus on the following sections and delete all other parts 

- [ ] Title - Please find which you want to do. check comments
- [ ] Goal - Check my comments in the goal . ReDo the goal . ask if you dont understand
- [ ] Problem statement 
- [ ] Research questions 
- [ ] paragraph heading of introduction (not the full paragraphs only what each paragraph will inform)

[Rnd_Rashid_Tajdeen_14_06_2022_reviewd.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/8935509/Rnd_Rashid_Tajdeen_14_06_2022_reviewd.pdf)

**Comment by deebuls** [comment]
@rashidahamedmeeran  Dont wait for last day of sprint to submit this . Make sure you give me something very week .

**Comment by rashidahamedmeeran** [comment]
[Rnd_Rashid_Tajdeen.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/9250766/Rnd_Rashid_Tajdeen_minimum.pdf)

**Comment by rashid-tajdeen** [comment]
[Rnd_Rashid_Tajdeen_proposal.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/10132570/Rnd_Rashid_Tajdeen_proposal.pdf)
For Dec 15th Submission.

**Comment by deebuls** [comment]
Do you want to go ahead with this rnd , because i have moved on with the PhD and Prof. is also not interested. You try to get a meeting with the Prof and see if he is interested then we can proceed . Or I would say you can look for other topics .

**Comment by rashid-tajdeen** [comment]
Yes I still would like to continue as i have worked on it for a while. I will try to get a meeting with professor and will let you know.

**Comment by rashid-tajdeen** [comment]
I talked to Prof. Dr. Sebastian Houben and explained the proposal. He said he would be the supervisor (but he would still have a final check with Prof. Nico if he is ok with it). Prof. Houben asked to send the proposal to him for reading and i have sent it to him.
 
[RnD_Proposal_Rashid_Tajdeen.pdf](https://github.com/RnDProjectsDeebul/RashidTajdeenRnD/files/10195341/RnD_Proposal_Rashid_Tajdeen.pdf)
